lib/eco_service_web/live/eco_service_live/edit_community_component.ex

```diff
@@ -0,0 +1,39 @@
+defmodule EcoServiceWeb.EcoServiceLive.EditCommunityComponent do
+  use EcoServiceWeb, :live_component
+
+  alias EcoService.EcoServiceContext
+
+  def render(assigns) do
+    ~H"""
+      <div>
+        <.simple_form :let={form} for={%{}} phx-target={@myself} phx-submit="update-community">
+          <.input field={form[:name]} label="Community Name" value={@community.name}/>
+          <.input field={form[:location_area_zone]} label="Location" value={@community.location_area_zone}/>
+          <.button>Save</.button>
+        </.simple_form>
+      </div>
+    """
+  end
+
+  def update(assigns, socket) do
+    community = EcoServiceContext.get_community_by_id(assigns.community_id)
+
+    {:ok,
+    socket
+    |> assign(:community, community)
+    }
+  end
+
+  def handle_event("update-community", params, socket) do
+    community = EcoServiceContext.update_community(socket.assigns.community, params)
+
+    case community do
```

Comment:
Why use `case` here at all? Is this code incomplete by any chance? I highly think it's incomplete. If you still think this code is complete, then, please have a look at the default generated code for a reference of best practice. If you're still unsure, then schedule a 10-15 minute meeting regarding this matter.

Reply:
I didn't match for error. Is that what you are trying to say?